index.bs

```diff
@@ -96,6 +96,25 @@ device running the user agent, and be accessed over a transport such as
 Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field
 Communications (NFC).
 
+## Specification Roadmap ## {#spec-roadmap}
+
+While many W3C specifications are directed primarily to audiences comprising user agent developers and web application developers (i.e., Web authors, herein also known as <em>[RP] developers</em>), the character of Web Authentication requires that this specification also be understood and properly used by those whose responsibilities include making decisions regarding:
+
+- [RP] web application login flows, account recovery flows, user account database content, etc. ([RP] developers)
+
+- Third-party web application frameworks / SDKs (Web framework developers)
+
+- OS platform API design and implementation in regards to platform-specific [=authenticator=] APIs, platform [=WebAuthn Client=] instantiation, etc. (OS platform developers)
+
+- [=Authenticator=] developers
+
+User agent developers, web framework developers, and OS platform developers ought to thoroughly understand the entire specification, except for perhaps the details of the various attestation statement formats ([[#defined-attestation-formats]]), since the latter are blobs emitted by authenticators that simply pass-through platforms and user agents on their way to the [RP] server (unless, of course, their platform emits an attestation statement of its own).
+
+[RP] developers (web authors) will find the [[#use-cases]], [[#sample-scenarios]], and [[#terminology]] sections useful for providing overall context. Further, the [[#api]], [[#rp-operations]], and [[#extension-predef]] sections provide the details underlying the aforementioned sections. [RP] server-side developers ought to undertand [[#cred-attestation]] in general. They will need to understand the latter in particular, as well as [[#defined-attestation-formats]], if they intend to verify credential attestations. 
```

Comment:
Typo: "undertand". Also should we add a sentence encouraging RPs to check attestation?